bookkeeper-benchmark/src/test/java/org/apache/bookkeeper/benchmark/TestBenchmark.java

```diff
@@ -38,6 +39,13 @@ public TestBenchmark() {
         super(5);
     }
 
+    @Before
+    public void setUp() throws Exception {
+        baseConf.setLedgerManagerFactoryClassName("org.apache.bookkeeper.meta.FlatLedgerManagerFactory");
```

Comment:
Why are you using a string and not an explicit reference to the class?
Having a strong compile time reference will ease refactoring and code analysis

Reply:
Because, if you reference the class,
```
[WARNING] /home/ivan/src/bookkeeper/checkouts/bench-flake/bookkeeper-benchmark/src/test/java/org/apache/bookkeeper/benchmark/TestBenchmark.java:[44,73] org.apache.bookkeeper.meta.FlatLedgerManagerFactory in org.apache.bookkeeper.meta has been deprecated
```

FlatLedgerManagerFactory is never going to move. That in itself would be a BC break.

Reply:
Okay

Reply:
just fyi, you can use full qualified import and `suppresswarnings("deprecation")` on the setup method.

Reply:
I'm well aware of suppressWarnings, but it was a small nit, that didn't add anything, the patch was already green with CI, was otherwise ready to go, just waiting on approvals and I wanted to get this change in asap, as master CI has been broken for 7 days now, and it's blocking other patches.

Reply:
yes, I am not blocking this PR. the comment was just "FYI"